src/UsgsAstroLsSensorModel.cpp

```diff
@@ -265,16 +265,7 @@ void UsgsAstroLsSensorModel::replaceModelState(const std::string &stateString )
    m_sunPosition = j["m_sunPosition"].get<std::vector<double>>();
    m_sunVelocity = j["m_sunVelocity"].get<std::vector<double>>();
 
-   m_logFile = j["m_logFile"].get<std::string>();
-   if (m_logFile.empty()) {
-     m_logger.reset();
-   }
-   else {
-     m_logger = spdlog::get(m_logFile);
-     if (!m_logger) {
-       m_logger = spdlog::basic_logger_mt(m_logFile, m_logFile);
-     }
-   }
+   m_logName = j["m_logName"].get<std::string>();

```

Comment:
I don't think we actually need the logName to be a member variable or a part of the state at all. There's no way for users to setup a new logger, so why would they specify a new logger name?